src/main/java/org/apache/sling/servlets/post/impl/operations/AbstractPostOperation.java

```diff
@@ -59,7 +61,7 @@ public abstract class AbstractPostOperation implements PostOperation {
     /**
      * Default logger
      */
-    protected final Logger log = LoggerFactory.getLogger(getClass());
+    protected final Logger log = LoggerFactory.getLogger(AbstractPostOperation.class);
```

Comment:
Again, why change the logger?

Reply:
This change is actually not desired, I would say, because now the log entries will always mention "AbstractPostOperation", instead of the inheritor?